homeassistant/components/onewire/quality_scale.yaml

```diff
@@ -0,0 +1,126 @@
+rules:
+  ## Bronze
+  config-flow:
+    status: todo
+    comment: Under review
+  test-before-configure: done
+  unique-config-entry:
+    status: done
+    comment: unique ID is not available, but duplicates are prevented based on host/port
+  config-flow-test-coverage:
+    status: todo
+    comment: Under review
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: entities do not subscribe to events
+  dependency-transparency:
+    status: todo
+    comment: The package is not built and published inside a CI pipeline
+  action-setup:
+    status: exempt
+    comment: No service actions currently available
+  common-modules:
+    status: done
+    comment: base entity available, but no coordinator
+  docs-high-level-description:
+    status: todo
+    comment: Under review
+  docs-installation-instructions:
+    status: todo
+    comment: Under review
+  docs-removal-instructions:
+    status: todo
+    comment: Under review
+  docs-actions:
+    status: todo
+    comment: Under review
+  brands: done
+
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: No service actions currently available
+  reauthentication-flow:
+    status: exempt
+    comment: Local polling without authentication
+  parallel-updates: done
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters:
+    status: todo
+    comment: Under review
+  docs-configuration-parameters:
+    status: todo
+    comment: Under review
+
+  ## Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery:
+    status: todo
+    comment: mDNS should be possible - https://owfs.org/index_php_page_avahi-discovery.html
+  stale-devices:
+    status: done
+    comment: Manual removal
```

Comment:
Can't we autodetect removed devices?

Reply:
> Can't we autodetect removed devices?

It is not possible to distinguish between a flaky device and a device that has been removed, so I felt it was better to do it manually.

Reply:
How many devices are flakey in general?

Reply:
On my system, it's the bus that is flaky so all of them :(